lib/elixir/pages/Guards.md

```diff
@@ -191,4 +191,33 @@ def foo(_other) do
 end
 ```
 
-The two forms are exactly the same semantically but there are cases where the latter one may be more aesthetically pleasing.
+For most cases the two forms are exactly the same, there exists a subtle difference in case of failing guards, as discussed in the section above.
+In case of a boolean expression guard, a failed element means the whole guard fails. In case of multiple guards it means the next one will be evaluates.
```

Comment:
We should keep the tense consistent. How about: `In case of multiple guards, it means the next one will be evaluated`?